src/stats.rs

```diff
@@ -0,0 +1,34 @@
+use std::borrow::Borrow;
+use api::{Id, Collection, Bool, OwnerId};
+
+#[cfg(feature = "unstable")]
+include!("stats.rs.in");
+
+#[cfg(not(feature = "unstable"))]
+include!(concat!(env!("OUT_DIR"), "/stats.rs"));
+
+request_ref! {
+    #[derive(Copy, Eq)]
+    struct Get for ["stats.get"](v => 5.44) -> Collection<Period> {
+        sized {
+            group_id: Option<Id> = () => {Option},
+            app_id: Option<Id> = () => {Option},
+        }
+        unsized {
+            date_from: str = ("") => {=},
+            date_to: str = ("") => {=},
```

Comment:
It seems the date format is fixed to `YYYY-MM-DD`, so it makes sense to either use custom type `struct Date { year: u32, month: u8, day: u8 }` or reuse [`NaiveDate`](https://lifthrasiir.github.io/rust-chrono/chrono/naive/date/struct.NaiveDate.html) from [chrono](https://crates.io/crates/chrono/). I propose to the latter, as it will make the library more interoperatable (given that I already have `chrono` as a dependency, we can just re-import `NaiveDate`, so library users won't have to depend on `chrono` directly).